liteflow-labs/starter-kit
# Add collection name to the Token page header + Refresh metadata functionality

### Project organization

- Trello card: https://trello.com/c/xIgYx9xZ
- Trello card: https://trello.com/c/FV5FMleK

### Description

- Added collection name on the home page featured Token
- Added collection name on the detail page of a Token
- Added the "Refresh Metadata" button and functionality to the detail page of a Token (in the ... menu)

### How to test

Check if the collection name appears on the Home page + Detail page
Check if the metadata refresh successfully
